integration-tests/python/README.md

```diff
@@ -0,0 +1,33 @@
+# Integration tests by Python
+
+This is a set of Python code that tests StellarStation API aiming to show how you can write API clients 
+for StellarStation API.
+
+Since these code are intended to server as sample code of the API, tests will not pass in your environment.
```

Comment:
Since this code is intended to show real world usage of the API, you will need to make some changes if you want the tests to pass in your environment:
- You need to change satellite ID and ground station ID in the code to match the satellites and ground stations that you own.
- If you don't own any satellites or ground stations, you may need to disable the corresponding tests
- open-stream tests depend on command and telemetry messages, as well as server infrastructure, specific to our test environment